libpysal/graph/base.py

```diff
@@ -1484,6 +1484,42 @@ def build_travel_cost(cls, df, network, threshold, kernel=None):
         Returns
         -------
         Graph
+
+        Examples
+        ---------
+        >>> import geodatasets
+        >>> import geopandas as gpd
+        >>> import osmnx as ox
+        >>> import pandana as pdna
+        >>> from libpysal.graph import Graph
+
+        >>> # read an example geodataframe
+        >>> df = gpd.read_file(geodatasets.get_path("geoda Cincinnati")).to_crs(4326)
```

Comment:
Given the way this will get rendered, it may be easier to keep the comments out of the code blocks.

```suggestion
        Read an example geodataframe:

        >>> df = gpd.read_file(geodatasets.get_path("geoda Cincinnati")).to_crs(4326)
```

This way you get text-code_block-text-code_block sequence in the docs. Otherwise you just get code_block-code_block which is not as friendly.

Reply:
thanks i never know how to do this